packages/react-layouts/src/Tabs/TabPanel/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Quid, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @flow
+import React from 'react';
+import styled from '@emotion/styled/macro';
+import { sizes } from '@quid/theme';
+
+export type Props = {
+  label: string,
+  id: string,
+  className?: string,
+};
+
+const TabPanel = styled(({ label, id, className, ...props }: Props) => (
```

Comment:
Here you are actually stropping out the Emotion generated className. I see the same thing happening in other files of this PR.

Reply:
Right, sorry.